config/README.md

```diff
@@ -23,6 +23,24 @@ The config files contain a features object that can be used to determine whether
 }
 ```
 
+Please make sure to add new feature flags alphabetically so they are easy to find and add any new feature flags in all config files (client.json, development.json, production.json, horizon.json, stage.json, test.json, wpcalypso.json).
```

Comment:
One more thing: It is not clear to me at what point a flag should be enabled in the `test.json`. A number of feature flags that are enabled in `production.json` are not mentioned in `test.json` at all. 🤔 

Maybe we could clarify a bit on `test.json` usage as well.

Reply:
It looks like the `test.json` config file was added back in #3793 to speed up tests by parallelizing.

Perhaps we can rely on @cometgrrl and her team for some guidance here?

Reply:
I'll merge for now and we can clarify later perhaps.

Reply:
Also asking here p1656924873116109-slack-C1A1EKDGQ.

Reply:
I replied to @yansern on the thread but the gist is that I am not aware of `test.json` in the context of end-to-end tests. 

Since the PR referenced is quite old (over 5 years ago), my guess is that `test/e2e/test.json` was the repo at some point but removed in favor of subsequent test parallelization efforts that led up to the use of [Magellan](https://github.com/TestArmada/magellan) until Selenium/WebDriverJS was deprecated earlier this year.